models/stg_facebook_ads__demographics_country.sql

```diff
@@ -0,0 +1,45 @@
+{{ config(enabled=var('ad_reporting__facebook_ads_enabled', True) and var('facebook_ads__using_demographics_country', False)) }}
+
+with base as (
+
+    select * 
+    from {{ ref('stg_facebook_ads__demographics_country_tmp') }}
+),
+
+fields as (
+
+    select
+        {{
+            fivetran_utils.fill_staging_columns(
+                source_columns=adapter.get_columns_in_relation(ref('stg_facebook_ads__demographics_country_tmp')),
+                staging_columns=get_demographics_country_columns()
+            )
+        }}
+        {{ fivetran_utils.source_relation(
+            union_schema_variable='facebook_ads_union_schemas', 
+            union_database_variable='facebook_ads_union_databases') 
+        }}
+    from base
+),
+
+final as (
+    
+    select 
+        source_relation, 
+        _fivetran_id as country_id,
+        country,
+        date as date_day,
+        cast(account_id as {{ dbt.type_bigint() }}) as account_id,
```

Comment:
No necessary to change for this PR since I know this decision was made in other models and we need to conform with the approach. However, it's interesting we make the decision to cast these IDs as integers. Usually we cast as string to be safe.

Reply:
Yeah I thought so too